src/indexer/services/contract/index.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Bridge as BridgeType,
+  BasicFeeHandler as BasicFeeHandlerType,
+  FeeHandlerWithOracle as FeeHandlerWithOracleType,
```

Comment:
@nmlinaric `DynamicFeeHandler` is the old `FeeHandlerWithOracle`?

Reply:
`DynamicFeeHandler` is an abstract contract that all "dynamic" fee handlers inherit, the new name is `DynamicERC20FeeHandlerEVM`

Reply:
Got it